src/metabase/public_settings.clj

```diff
@@ -9,112 +9,106 @@
+  (tru "The map tile server URL template used in map visualizations, for example from OpenStreetMaps or MapBox.")
   :default "http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png")
 
 (defsetting enable-public-sharing
-  "Enable admins to create publically viewable links (and embeddable iframes) for Questions and Dashboards?"
+  (tru "Enable admins to create publically viewable links (and embeddable iframes) for Questions and Dashboards?")
   :type    :boolean
   :default false)
 
 (defsetting enable-embedding
-  "Allow admins to securely embed questions and dashboards within other applications?"
+  (tru "Allow admins to securely embed questions and dashboards within other applications?")
   :type    :boolean
   :default false)
 
 (defsetting enable-nested-queries
-  "Allow using a saved question as the source for other queries?"
+  (tru "Allow using a saved question as the source for other queries?")
   :type    :boolean
   :default true)
 
 
 (defsetting enable-query-caching
-  "Enabling caching will save the results of queries that take a long time to run."
+  (tru "Enabling caching will save the results of queries that take a long time to run.")
   :type    :boolean
   :default false)
 
 (defsetting query-caching-max-kb
-  "The maximum size of the cache, per saved question, in kilobytes:"
+  (tru "The maximum size of the cache, per saved question, in kilobytes:")
   ;; (This size is a measurement of the length of *uncompressed* serialized result *rows*. The actual size of
   ;; the results as stored will vary somewhat, since this measurement doesn't include metadata returned with the
   ;; results, and doesn't consider whether the results are compressed, as the `:db` backend does.)
   :type    :integer
   :default 1000)
 
 (defsetting query-caching-max-ttl
-  "The absoulte maximum time to keep any cached query results, in seconds."
+  (tru "The absoulte maximum time to keep any cached query results, in seconds.")
   :type    :integer
   :default (* 60 60 24 100)) ; 100 days
 
 (defsetting query-caching-min-ttl
-  "Metabase will cache all saved questions with an average query execution time longer than
-   this many seconds:"
+  (tru "Metabase will cache all saved questions with an average query execution time longer than this many seconds:")
   :type    :integer
   :default 60)
 
 (defsetting query-caching-ttl-ratio
-  "To determine how long each saved question's cached result should stick around, we take the
-   query's average execution time and multiply that by whatever you input here. So if a query
-   takes on average 2 minutes to run, and you input 10 for your multiplier, its cache entry
-   will persist for 20 minutes."
+  (tru "To determine how long each saved question's cached result should stick around, we take the query's average execution time and multiply that by whatever you input here. So if a query takes on average 2 minutes to run, and you input 10 for your multiplier, its cache entry will persist for 20 minutes.")
```

Comment:
@senior what's the best way we can translate strings like this but maintain a sane line length?

Do we need to do
```clojure
(tru (str "..."
          "..."))
```

?

@mazameli taking something like this that was previously broken out into multiple lines and making it one single massive line makes the code hard to read here on GH. It would be nice if we could keep things broken out somehow so you don't need to scroll horizontally for weeks to read our code

Reply:
I agree on making it hard to view in Github, but AFAIK, that's just the way gettext works. It can't follow multiple string literals. This is ok:
```
(str (tru "My first sentence.")
     (tru "My second sentence."))
;;-> Works, creates 2 separate extracted strings
```
But it will silently only extract the first string in the below:
```
(tru (str "My first sentence."
          "My second sentence."))
;; -> Will miss "My second sentence"
```
I tired to generate the translation files from the project to get an example to paste in but that doesn't appear to be working. @tlrobinson how were you generating those when you were testing?

Reply:
Oh yeah, I forgot about that. Breaking something long up into multiple strings to translate sounds silly. I guess super-long lines will have to do

Reply:
@senior IIRC the general workflow is:

1. Tag strings
2. Run `./bin/i18n/update-translation`
3. Translate `.po` files
4. Run `./bin/i18n/build-translation-resources`

Reply:
^ BTW This is the long line in question